images/builder/Dockerfile

```diff
@@ -118,6 +118,9 @@ COPY --from=builder ["/barnacle", "/usr/local/bin/"]
 # add ecr login help
 COPY --from=builder ["/go/bin/docker-credential-ecr-login", "/usr/local/bin/"]
 
+# Install docker-compose
+RUN pip install docker-compose
```

Comment:
Could we perhaps pin this to a minor version? `pip` should have support for something like `~=x.y.0` if memory serves.

Reply:
I believe `>=1.24.0,<1.25.0` should work

Reply:
So would `~=1.24.0` (or `==1.24.*`)*, but whatever floats your boat ;)

\* [source](https://stackoverflow.com/a/50080281)